bootstrap.sh

```
@@ -1 +1,33 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Good: shebang is present and strict modes are enabled (`set -euo pipefail`). This meets the requirement for a POSIX shell script that fails fast on error.